.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -U tox tox-gh-actions
+    - name: Run tests
+      run: tox
+    - name: Upload coverage report
+      uses: codecov/codecov-action@v1
+      if: ${{ matrix.os == 'ubuntu-latest' && matrix.python-version == '3.8' }}
```

Comment:
can't you just upload them all with codecov and it unions them?

Reply:
I guess so, just seemed wasteful.

Reply:
```suggestion
```